kevm-pyk/pyproject.toml

```diff
@@ -12,6 +12,7 @@ authors = [
 
 [tool.poetry.dependencies]
 python = "^3.8"
+pathos = "*"
```

Comment:
Did you consider using standard library features (e.g. [multiprocessing](https://docs.python.org/3/library/multiprocessing.html))?

Reply:
Yes, I tried multiprocessing first. Most things I searched said that pathos is better behaved. multiprocessing using default Python pickling to create copies of the Python interpreter, which fails in many cases (and failed in this case). pathos.multiprocessing using dill instead, which handles pickling almost everything (and works in our case).

https://stackoverflow.com/questions/29121979/python-pathos-multiprocessing-vs-class-methods